packages/docs/pages/reference/elements/ad.mdx

```diff
@@ -8,12 +8,17 @@ import {
 
 # `<ad>`
 
-Short description of what the `<ad>` tag does.
+The self-closing `<ad/>` tag displays the abbreviation
+ for the Latin <em>anno Domini</em>.
 
 ## Syntax
 
 <SyntaxDisplay name="ad" />
 
 ## Examples
 
-    
\ No newline at end of file
+ ```ptx-example
+<FRAGMENT parents="p">
+    Math XML became PreTeXt in 2017<ad/>.
```

Comment:
Can you double check that the intended usage of this tag has no space between the date and the `<ad/>`?